enclave_manager/Dockerfile

```diff
@@ -230,6 +230,7 @@ ARG SGX_MODE
 ARG MAKECLEAN
 ARG TCF_DEBUG_BUILD
 ARG ENCLAVE_TYPE
+ARG ENCLAVE_BUILD_TYPE
 
```

Comment:
Is `ENCLAVE_BUILD_TYPE` getting used ?

Reply:
Nope, only ENCLAVE_TYPE is being use. deleted it.